beep/helpers/featurizer_helpers.py

```diff
@@ -406,50 +493,147 @@ def get_V_I(df):
     result['current'] = current
     return result
 
-
-def get_v_diff(diag_num, processed_cycler_run, soc_window):
+def get_v_diff(processed_cycler_run, diag_pos, soc_window):
     """
-    This function helps us get the feature of the variance of the voltage difference
-    across a specific capacity window
-    Argument:
-            diag_num(int): diagnostic cycle number at which you want to get the feature, such as 37 or 142
-            processed_cycler_run(process_cycler_run object)
-            soc_window(int): let the function know which step_counter_index you want to look at
+    This function calculates voltage difference between a hppc cycle we are interested (at diag_pos) and the initial
+    diag cycle (cycle 2) for a specific soc range
```

Comment:
For the sake of formality, it would help to streamline the language in all docstrings. For instance, this could be "Method to calculate voltage difference between a hppc cycle at a position specified by <diag pos> and the first one."